apps/rush-lib/src/cli/logic/InstallManager.ts

```diff
@@ -469,6 +464,14 @@ export default class InstallManager {
     const commonPackageJsonFilename: string = path.join(this._rushConfiguration.commonTempFolder,
       RushConstants.packageJsonFilename);
 
+    if (shrinkwrapFile) {
+      // Resync the temporary shrinkwrap file.
+      // Copy (or delete) common\npm-shrinkwrap.json --> common\temp\npm-shrinkwrap.json
+      shrinkwrapFile.save(this._rushConfiguration.tempShrinkwrapFilename);
```

Comment:
>save [](start = 21, length = 4)

Does the file timestamp matter?

Reply:
No

---
In reply to: [170407274](https://github.com/Microsoft/web-build-tools/pull/543#discussion_r170407274) [](ancestors = 170407274)